tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -1203,8 +1203,6 @@ public function createContributionWithTax() {
     $form->testSubmit(array(
        'total_amount' => 100,
         'financial_type_id' => $financialType['id'],
-        'receive_date' => '04/21/2015',
```

Comment:
can we update the format on these dates rather than remove?

Reply:
@eileenmcnaughton 2 reasons why I removed it:
1. Provided the default value of current date time in the desired format [here](https://github.com/civicrm/civicrm-core/pull/10841/files#diff-0573f78042d230ffc1d5e61c1749ae77R1291)
2. In these unit-tests receive date has no importance, however I have retained in some UT like [here](https://github.com/civicrm/civicrm-core/pull/10841/files#diff-4cf264b86743b01490484978a3bdd2d8R1690) where it's needed.

Or do you still want to update their formats?

Reply:
that seems fine - sounds like we are not losing any test data